mkdocs.yml

```diff
@@ -33,21 +33,21 @@ markdown_extensions:
 nav:
   - Quick start: index.md
   - Examples: examples.md
-  - Image acquisition:
-    - Image acquisition: image-acquisition/image-acquisition.md
-    - Image files: image-acquisition/image-file.md
-    - Video files: image-acquisition/video-file.md
+  - Concepts: concepts.md
+  - Data Management: metadata.md
```

Comment:
I think we dropped `Concepts`, right?

Also, would you mind moving `Data Management` entry to after the `Running Inferences` part (before `APIs`)? I feel that this section is not as important as acquiring image and running predictions, and most developers will not use this, so it should be presented only when the user has deeper knowledge about the SDK and the platform.